mantidproject/mantid
# Create indirect data manipulation interface

**Description of work.**

This PR creates a new interface called Indirect Data Manipulation, it's function is to contain the elements currently within data analysis and data reduction that do not handle raw data and do not fit (symmtrise, Sqw, Moments, Elwin, Iqt). this PR only implements symmtrise, Sqw and Moments into the interface (those from indirect data reduction) Elwin and Iqt will be implemented seperatly as they require further changes.

**To test:**

1. Open Indirect Data Analysis, check that symmtrise, Sqw and Moments are no longer present.
2. Open the Indirect Data Manipulation Interface it should contain symmtrise, Sqw and Moments tabs.
3. In each tab run through it's workflow and check they output correctly.
5. close the interface and open the Indirect Settings and change your facility.
6. reopen the manipulation interface and in the s(q,w) tab check that the instrument option have changed.

Fixes #33676 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
